app/assets/stylesheets/custom.css.scss

```diff
@@ -178,13 +178,13 @@ html[xmlns].clearfix {
       display: block;
       text-decoration: none;
     }
-    .check-card-form {
-      width: 360px;
-      margin: 15px auto 0;
-      .field {
-        width: 240px;
-        height: 20px;
-      }
+  }
+  .review-form {
+    width: 360px;
+    margin: 15px auto 0;
+    .field {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3